### PR TITLE
last tweaks for 30th

### PR DESCRIPTION
**Pull request by towardsnewtech**
update mobile design on roadmap page
update twitter button link on claim success page
update twitter button text
update claim page approving start speed
remove greeting sentence

### CODE REVIEW

File: components/ClaimPage/ClaimPage.tsx [hunk 1]
```diff
@@ -48,9 +48,9 @@ const ClaimPage = () => {
   const handleBurn = async () => {
     if (!signer) return
     try {
+      setModalStatus(ModalStatus.APPROVING)
```
Comment (sweetmantech) [inline components/ClaimPage/ClaimPage.tsx:51]: ⚡

File: components/RoadmapPage/RoadmapPage.tsx [hunk 2]
```diff
@@ -25,14 +24,16 @@ const RoadmapPage = () => {
...
+
     stages.map((stage: StageData, index: number) => {
       // eslint-disable-next-line react-hooks/exhaustive-deps
       if (index === 0) offset = Math.abs(new Date(stage.date).getTime() - new Date().getTime())
 
       const currentOffset = Math.abs(new Date(stage.date).getTime() - new Date().getTime())
 
       if (offset > currentOffset) setCurrentIndex(index)
-      offset = Math.abs(new Date(stage.date).getTime() - new Date().getTime())
+      offset = currentOffset
```
Comment (sweetmantech) [inline components/RoadmapPage/RoadmapPage.tsx]: what is `offset` being used for after `line 35`?

can we just delete this line?

**Review by SweetmanTech — APPROVED** [review]
ship it

thanks for improving page loads ⚡

**Review by SweetmanTech — APPROVED** [review]
ship it 🚢

**Pull request merged**